test/test-browsers.js

```diff
@@ -66,7 +66,16 @@ function processData(data, displayBrowsers, requiredBrowsers, category, logger,
   }
   for (const key in data) {
     if (key === "__compat") continue;
-    hasErrors |= processData(data[key], displayBrowsers, requiredBrowsers, category, logger, (path && path.length > 0) ? `${path}.${key}` : key);
+    hasErrors = processData(
+      data[key],
+      displayBrowsers,
+      requiredBrowsers,
+      category,
+      logger,
+      (path && path.length > 0)
+        ? `${path}.${key}`
+        : key,
+    ) || hasErrors;
```

Comment:
This prevents `hasErrors` from being converted into&nbsp;a&nbsp;`number`.

Reply:
Thanks, can we add that as a code comment? I think it would help understanding this line in the future.